testrunner-maven-plugin/src/main/scala/edu/illinois/cs/testrunner/mavenplugin/TestRunner.scala

```diff
@@ -3,6 +3,8 @@ package edu.illinois.cs.testrunner.mavenplugin
 import java.nio.file.{Files, Paths}
 
 import edu.illinois.cs.testrunner.configuration.Configuration
+import edu.illinois.cs.testrunner.data.framework.TestFramework
```

Comment:
Why is the TestFramework import necessary?

Reply:
Oh, yea. It's redundant. Deleted.